Twelve Days: add instructions.append and hints (#3050)
exercism/python: exercises/practice/twelve-days/.docs/hints.md

# Hints

# General

> Have you tried to break your logic up into one or more helper functions?

Reviewer: ```suggestion
Have you tried to break your logic up into one or more helper functions?
This type of song is known as a [cumulative song][cumulative song], so it could be particularly well suited to being "broken up".
```

> Doing so may make your code easier to reason about and will conform to the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).

Reviewer: ```suggestion
Doing so may make your code easier to reason about and will conform to the [Single Responsibility Principle][Single Responsibility Principle].
```

# Approaches

## Looping

> To learn about the ways to loop in Python, please see the [docs](https://wiki.python.org/moin/ForLoop).

Reviewer: ```suggestion
To learn about the ways to loop in Python, please see [loops](https://exercism.org/tracks/python/concepts/loops).
```
